package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-new-file",
-  "version": "3.2.2",
+  "version": "4.0.1",
```

Comment:
4.0.2?

Reply:
Ok I did that change in both `package.json` & `package-lock.json`